Source/Plugins/NetworkEvents/NetworkEvents.cpp

```diff
@@ -238,10 +245,72 @@ String NetworkEvents::handleSpecialMessages(const String& s)
         status += CoreServices::RecordNode::getExperimentNumber();
         return status;
     }
+    else if (cmd.compareIgnoreCase("TTL") == 0)
+    {
```

Comment:
Implemented the first comment. Unsure about whether the second is totally needed? I think it still processes events that are in the queue even when acquisition is stopped.

Reply:
Yeah, the second part isn't totally needed; it could be a later enhancement. If we're not adding new events while acquisition is stopped, then events not being processed in time and staying in the queue from one stop to the next start would just be an edge case.

It wouldn't process events in the queue when acquisition is stopped though, since the queue is being popped in the `process` function, which only runs when acquisition is on. Unless I'm missing something.